### DISCUSSION TITLE
Interest in Solving Issues #838 and #840

### DISCUSSION THREAD

**Discussion by Ravish990** [neurobagel/annotation_tool#843] [General]
Hello @rmanaem @alyssadai  @yarikoptic  @surchs  @jarmoza,

I am Ravish. I am currently looking into the issues and have identified that I can solve issues #838 and #840. I have explored these issues in detail and am confident that I can address them effectively. Could you please assign them to me?

Thank you for your consideration !

**Comment by rmanaem** [comment]
Hi @Ravish990, thanks for your interest in contributing to Neurobagel! We are currently reworking the annotation tool so I think these issues or even the annotation tool is not a great place to start. If you are interested in frontend, you can also take a look at our query tool: https://github.com/neurobagel/query-tool
